src/page/addItems/FileInput.js

```diff
@@ -0,0 +1,17 @@
+function FileInput() {
+  return (
+    <div>
+      <div className="register-title">
+        <h2>상품 등록하기</h2>
+        <button className="small-button">등록</button>
+      </div>
+      <div className="product-input">
+        <p className="file-input-title">상품 이미지</p>
+        <label className="file-input-label" for="input-file" />
+        <input type="file" id="input-file"></input>
```

Comment:
```suggestion
        <input type="file" id="input-file" />
```

위에 라벨에서 해주신것처럼 children 이 없을 때는 이렇게 해주기도 합니다!